packages/init2/src/lib/listr.ts

```diff
@@ -0,0 +1,15 @@
+// @ts-expect-error - @types package not available
+import UpdateRenderer from "@lihbr/listr-update-renderer";
```

Comment:
You don't trust me as a maintainer? 😥

jk, I think we might just have it in `lib` here since it's a small package (just one file), or fork it if we plan on using it a multiple places indeed.

Reply:
lol, I trust you. I'm just looking out for you and the company.

`lib` sounds good to me if it's small enough to fit in a single file.